Backend/Remora.Discord.Rest/StaticTokenStore.cs

```diff
@@ -20,30 +20,34 @@
 //  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 //
 
+using System.Threading;
+using System.Threading.Tasks;
 using JetBrains.Annotations;
 
 namespace Remora.Discord.Rest;
 
 /// <summary>
-/// Represents a storage class for a single token.
+/// Represents a storage class for a static token.
 /// </summary>
 [PublicAPI]
-public class TokenStore : ITokenStore
+public class StaticTokenStore : IAsyncTokenStore
 {
+    private readonly ValueTask<string> _token;
```

Comment:
`ValueTask` instances should generally only be awaited once, since its underlying semantics are different from a normal task. See https://devblogs.microsoft.com/dotnet/understanding-the-whys-whats-and-whens-of-valuetask/#valid-consumption-patterns-for-valuetasks for an explanation.

A simple fix would be to just store the string and create a new ValueTask on each invocation.